### PR TITLE
Issue #27 fix

### LINKED ISSUE #27: Frame display not aligned with the UAV real front direction
The displayed frame on the pygame window does not match the orientation of the real Tello, it seems to constantly "look left".
Some data is probably lost during the image processing procedure.

### LINKED ISSUE #30: Video output crashes after a random amount of time using the real Tello
````
  File "C:\Users\pierre.bagnara\PycharmProjects\Challenge-drone-EA\DJITelloPy\djitellopy\tello.py", line 1073, in update_frame
    for frame in self.container.decode(video=0):
  File "av\container\input.pyx", line 186, in decode
  File "av\container\input.pyx", line 146, in demux
  File "av\container\core.pyx", line 292, in av.container.core.Container.err_check
  File "av\error.pyx", line 336, in av.error.err_check
av.error.OSError: [Errno 5] I/O error: 'udp://0.0.0.0:11111'; last error log: [h264] error while decoding MB 52 29, bytestream -6
````

### PR DESCRIPTION
**Pull request by pbag47**
The previous pull request merge did not work as intended and the previous issue #27 fix has been deleted

### CODE REVIEW

**Comment by pbag47** [comment]
Refactored archiecture (once again)

The main thread runs the setup, then runs the pygame loop (user input detection + pygame window update) until the stop signal, then runs the stop procedure.
A separate thread is in charge of the reception of frames coming from the Tello. It puts them in a Last-in First-out queue to make them available to other threads.
Another separate thread handles the image processing. A loop takes the most recent frame out of the LiFo queue and dismisses the outdated ones. It then detects markers and controls the UAV repeatedly until the stop signal.

This architecture allows for extensive image-processing features (or any other time-consuming computations) in real time without deteriorating the UAV responsiveness.

The simulator seems a bit laggier than before, but the real Tello is way better handled.
(Tested in manual control mode with an xbox gamepad in the SMO6 flight zone)

File: main.py [hunk 1]
```diff
@@ -42,39 +47,83 @@ def init_env() -> (Tello, BackgroundFrameRead):
...
-        TelloActuators.run(RCStatus)
-        # Update pygame display window
-        variables_to_print = parameters.merge_dicts([TelloSensors.__get_dict__(),
-                                                     ModeStatus.__get_dict__(),
-                                                     RCStatus.__get_dict__(),
-                                                     marker_status.__get_dict__()])
-        Display.run(frame, variables_to_print)
-    else:
-        print('Frame received before the end of setup, unable to process it yet -> Frame dismissed')
+class ImageProcess:
```
Comment (Archimonde666) [inline main.py:50]: I like to have classes in separated files

File: parameters.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 class ENV:
     REAL: int = 0
     SIMULATION: int = 1
-    status: int = SIMULATION
+    status: int = REAL
```
Comment (Archimonde666) [inline parameters.py:26]: Soon i create and a configuration file that is stored locally only and gitignored to overcome that problem

File: subsys_tello_sensors.py [hunk 3]
```diff
@@ -69,3 +59,35 @@ def __get_dict__(cls) -> dict:
                          'Pitch': cls.pitch,
                          'Yaw': cls.yaw}
         return sensors
+
+
+class FrameReader:
```
Comment (Archimonde666) [inline subsys_tello_sensors.py:64]: same remark : separate file ?

**Review by Archimonde666 — APPROVED** [review]

**Pull request merged**